src/webgpu/api/validation/image_copy/texture_related.spec.ts

```diff
@@ -112,9 +111,12 @@ TODO: update to test all texture usages`
       ] as const)
       .beginSubcases()
       .combine('usage', [
+        GPUConst.TextureUsage.COPY_SRC | GPUConst.TextureUsage.COPY_DST,
         GPUConst.TextureUsage.COPY_SRC | GPUConst.TextureUsage.TEXTURE_BINDING,
+        GPUConst.TextureUsage.COPY_SRC | GPUConst.TextureUsage.STORAGE_BINDING,
         GPUConst.TextureUsage.COPY_DST | GPUConst.TextureUsage.TEXTURE_BINDING,
-        GPUConst.TextureUsage.COPY_SRC | GPUConst.TextureUsage.COPY_DST,
+        GPUConst.TextureUsage.COPY_DST | GPUConst.TextureUsage.STORAGE_BINDING,
+        GPUConst.TextureUsage.TEXTURE_BINDING | GPUConst.TextureUsage.STORAGE_BINDING,
```

Comment:
I think the main thing we wanted to test here is that none of the other usages *also* allow copies. For example TEXTURE_BINDING or STORAGE_BINDING shouldn't allow it to be copied from, and STORAGE_BINDING or RENDER_ATTACHMENT shouldn't allow it to be copied to.

So in principle I think all we really need is the values of `kTextureUsages`, though the extra cases are good too (especially COPY_SRC | COPY_DST).
Could do something like
```
[
...kTextureUsages,
COPY_SRC | COPY_DST,
TEXTURE_BINDING | STORAGE_BINDING | RENDER_ATTACHMENT,
]
```

This will also mean the test gets automatically enhanced in the future when we update `kTextureUsages`.

Reply:
Thank you for suggesting it. Then, how about using two combines like the below?

  `.combine('usage0', kTextureUsages)`
  `.combine('usage1', kTextureUsages)`

Then, it looks like we can test more combined usages.

Reply:
Great idea. Looks good to me.